automation/roles/add-repository/tasks/main.yml

```diff
@@ -125,45 +124,43 @@
 
     # Enable Debuginfo repository
     - block:
-        - name: Enable PostgreSQL debuginfo repository
-          ansible.builtin.shell: |
-            set -o pipefail;
-            sed -i '/\[pgdg[0-9]*-debuginfo\]/,/^$/ s/enabled=0/enabled=1/' {{ pgdg_redhat_repo_path }}
-            sed -i '/\[pgdg[0-9]*-debuginfo\]/,/^$/ s/gpgcheck=1/gpgcheck=0/' {{ pgdg_redhat_repo_path }}
-          vars:
-            pgdg_redhat_repo_path: "/etc/yum.repos.d/pgdg-redhat-all.repo"
+        # - name: Enable PostgreSQL debuginfo repository
+        #   ansible.builtin.shell: |
+        #     set -o pipefail;
+        #     sed -i '/\[pgdg[0-9]*-debuginfo\]/,/^$/ s/enabled=0/enabled=1/' {{ pgdg_redhat_repo_path }}
+        #     sed -i '/\[pgdg[0-9]*-debuginfo\]/,/^$/ s/gpgcheck=1/gpgcheck=0/' {{ pgdg_redhat_repo_path }}
+        #   vars:
+        #     pgdg_redhat_repo_path: '/etc/yum.repos.d/pgdg-redhat-all.repo'
```

Comment:
I think we could use the condition for ansible_architecture here, instead of disabling the task completely.

Reply:
Yeah surely, this was just a quick & dirty win to get started.